README.md

```diff
@@ -3,3 +3,11 @@
 Install pyenv, and it should automagically use the python version specified in `.python-version`
 
 Tooling is listed in dev-requirements.txt
+
+## Run Development Server
+
+`docker compose -f docker-compose.local.yml up`
```

Comment:
Don't both of these need to be run in the `admin/cincoctrl/` directory? Which has it's own slightly different instructions for docker setup. Is there a reason to have those in to places?

Reply:
This should get moved into the README located at `admin/cincoctrl/` - this is in fact how to run the development server though, even though it doesn't exist in the README there. Will address is upcoming reorg PR.